krux_cloud_health/CloudAPI.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+#
+# © 2015 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import pprint
+import urlparse
+import json
+
+#
+# Third party libraries
+#
+
+import requests
+
+#
+# Internal libraries
+#
+
+from krux.logging import get_logger
+from krux.stats import get_stats
+from krux.cli import Application, get_group
+
+NAME = 'cloud-API'
+API_ENDPOINT = "https://apps.cloudhealthtech.com/olap_reports/"
+
+
+class CloudAPI(Application):
+
+    def __init__(self, name=NAME):
+        # Call to the superclass to bootstrap.
+        super(CloudAPI, self).__init__(name=name)
+
+        self.api_key = self.args.api_key
+        self.logger = get_logger(name)
+
+    def add_cli_arguments(self, parser):
+        # Call to the superclass first
+        super(CloudAPI, self).add_cli_arguments(parser)
+
+        group = get_group(parser, self.name)
+
+        group.add_argument(
+            '--name',
+            help="Name of user"
+        )
+
+        group.add_argument(
+            '--api-key',
+            help="API key to retrieve data",
+        )
+
+    def get_report(self, report, api_key):
+        uri = urlparse.urljoin(API_ENDPOINT, report)
+        # uri += "?" + time_filter
+        uri += "?api_key=%s" % api_key
```

Comment:
Nit: `requests` has a way to handle query strings: http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

This is not a show stopper for this PR, but we should fix this before production.